src/validators/mod.rs

```diff
@@ -124,6 +131,12 @@ pub fn create_issues(input: &str, max_issues: usize) -> Response {
     }
 }
 
+pub fn create_issues_post(body: web::BytesMut, max_issues: usize) -> Response {
```

Comment:
isn't this more meant to be in `daemon.rs` ? I'd prefer for all the actix stuff to be there

Reply:
is it really actix stuff ? It’s reading a gtfs from a bunch of bytes (the bytes being defined in actix, it’s true)

Reply:
since it's defined in actix i'd say yes.

i'm fine to have a function here  `create_issues_from_reader<T: std::io::Read + std::io::Seek>(reader: T, max_issues:usize)` though (with the cursor being created in the actix part though)